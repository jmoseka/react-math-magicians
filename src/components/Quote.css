body {
  height: 100vh;
}

.container-quote {
  height: 70vh;
  display: flex;
  justify-content: center;
  align-items: center;
}
