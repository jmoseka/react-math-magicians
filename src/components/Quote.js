import './Quote.css';

const Quote = () => (
  <div className="container container-quote">
    <blockquote>
      <p className="h5">
        {`Mathematics is not about numbers, equations, 
              computations, or algorithms: it is about understanding. -William Paul Thurston`}
      </p>
    </blockquote>
  </div>
);

export default Quote;
