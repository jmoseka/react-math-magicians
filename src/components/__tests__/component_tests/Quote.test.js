import '@testing-library/jest-dom/extend-expect';
import { cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../../Quote';

afterEach(() => {
  cleanup();
});

test('should match snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
