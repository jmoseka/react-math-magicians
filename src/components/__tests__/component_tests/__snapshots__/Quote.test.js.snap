// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="container container-quote"
>
  <blockquote>
    <p
      className="h5"
    >
      Mathematics is not about numbers, equations, 
              computations, or algorithms: it is about understanding. -William Paul Thurston
    </p>
  </blockquote>
</div>
`;
